docs/search/optimize-search-partitions.md

```diff
@@ -26,12 +26,12 @@ Consider the following queries:
 
 * **Query 1**: There are no custom Partitions created and you only have the Default Index, 100% of your data is scanned in order to find all production log messages for the Snort security app.
```

Comment:
100% of your data across all partitions is scanned in order to find ....

Reply:
updated